client/src/components/Login.style.js

```diff
@@ -0,0 +1,85 @@
+import styled from 'styled-components';
+
+export const StyleAll = styled.div`
+  background-color: #f9d52f;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-flow: column wrap;
+`;
+
+export const Title = styled.h1`
+  fontfamily: Work Sans;
+  fontstyle: Black;
+  fontsize: 38px;
+  background: white;
+  margin: 50px;
+`;
+export const StyledForm = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-flow: column wrap;
+`;
+export const InputContainer = styled.div``;
+export const StyledLabel = styled.label`
+  font-size: 18px;
+  text-align: center;
+  color: black;
+  font-weight: bold;
+`;
+export const StyledText = styled.h1`
+  font-size: 18px;
+  text-align: center;
+  color: black;
+`;
+export const StyledInput = styled.input`
+  display: block;
+  background: #ffffff;
+  border-radius: 10px;
+  margin-bottom: 30px;
+  width: 200px;
+  padding: 10px 0px;
+`;
+export const StyledBottom = styled.div``;
```

Comment:
Can be removed

Reply:
No, when deleting it from `login.style.js` will effect for `login.js` and it shows an error `styleBottom is not defined in login.style.js`